### ISSUE TITLE
Tree shake-able badge

### ISSUE THREAD

**Issue by conartist6** [pastelsky/bundlephobia#224]
I maintain a functional library (iter-tools). A main selling point is that it is tree shake-able, since it exports quite a lot of methods and most people only need a few. I'd love to advertise this fact with a badge that links to bundlephobia, and which serves as a data display for the tree-shakeable icon(s) that show up on the bundlephobia website itself. I propose a new endpoint of the form:
```
/api/tree-shaking =>
{
  "tree-shakeable": true,
  "side-effects": false,
}
```
The badge itself would be implemented on shields.io, and there would be some further design work to figure out how best to combine those two pieces or information.

I'm happy to code this up and submit a PR. Are there any concerns or suggestions relating to this proposal?

**Comment by marcofugaro** [comment]
Yes! This would also be useful to show in the single package page on the website! The correct flag to check is the `sideEffect` property fo the package.json.

https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free
